erpnext/stock/doctype/purchase_receipt/purchase_receipt.py

```diff
@@ -838,3 +832,20 @@ def make_work_orders(items, purchase_receipt, company, project=None):
 		out.append(work_order)
 
 	return [p.name for p in out]
+
+@frappe.whitelist()
+def get_bom_query(doctype, txt, searchfield, start, page_len, filters):
```

Comment:
```suggestion
def get_bom_query(doctype, filters):
```

Reply:
As the function is called in get_query method definition requires this, it gives error without this args